### ISSUE TITLE
VariableDeclarationPerScopeConventions -> requireInTheBeginning reports functions without any variables

### ISSUE THREAD

**Issue by benthemonkey** [dsheiko/jscodesniffer#12]
If I set `requireInTheBeginning: true`, I expected 0 or 1 occurrences of the `var` keyword as the first line of a function would be accepted. However, jscodesniffer is reporting an error for code like the following (sorry but I had difficulties making a mocha test case):

``` js
var noVariables = function(x) {
    return x;
};
```

Is this a bug or intentional?
